server/models.py

```diff
@@ -260,6 +260,7 @@ class Status(models.TextChoices):
         models.PositiveIntegerField()
     )  # For Cross Pool and Brackets to have round number
 
+    name = models.CharField(max_length=20, null=True, blank=True)
```

Comment:
The test of the fixture is a separate issue, #163

Reply:
> We would need to update the `sample_data.json`, I think... to include the name field. May be we should add a test to detect such issues in future PRs, where the fixture is not updated.

the match names are generated while creating the pools and brackets, so we'd have to run those again to get the match names right ?

Reply:
There are two separate issues, here. 

1. `python manage.py loaddata server/fixtures/sample_data.json` fails, after running the latest migrations on this PR because the name field has no value in the fixture. You need to update the fixture file. To do this, you can dump the data in your current DB (assuming there's not much new data) than what is already in the fixture. Or, switch to main, remove the DB, run migrations, switch to current PR branch, run migrations, dump data.   
3. The other issue would be to ensure that the match names for existing Matches in the DB is correct. For this, you'd  need to write a data migration with runs the code that generates the names.  You can see migrations `0039` and `0032` for examples of this. If you need more help with this, feel free to ping on Zulip with questions.

Reply:
yeah, i thought we'll leave the match names to be blank for older tournaments, but since we're changing the UI to display the names from DB, we'd have to generate the names. I'll ask doubts on zulip if i have any, thanks punch

Reply:
> `python manage.py loaddata server/fixtures/sample_data.json` fails, after running the latest migrations on this PR because the name field has no value in the fixture. You need to update the fixture file. To do this, you can dump the data in your current DB (assuming there's not much new data) than what is already in the fixture. Or, switch to main, remove the DB, run migrations, switch to current PR branch, run migrations, dump data.

@SibiAkkash I think you can ignore this! It seems like the migration hadn't run locally for me, and that was the cause of the failure. Not an issue with the fixture and the new field added to the model.

Reply:
also, once we have a new data migration, should we have some script that dumps the data into the sample_data.json file ?

Reply:
> There are two separate issues, here.
> 
> 1. `python manage.py loaddata server/fixtures/sample_data.json` fails, after running the latest migrations on this PR because the name field has no value in the fixture. You need to update the fixture file. To do this, you can dump the data in your current DB (assuming there's not much new data) than what is already in the fixture. Or, switch to main, remove the DB, run migrations, switch to current PR branch, run migrations, dump data.
> 2. The other issue would be to ensure that the match names for existing Matches in the DB is correct. For this, you'd  need to write a data migration with runs the code that generates the names.  You can see migrations `0039` and `0032` for examples of this. If you need more help with this, feel free to ping on Zulip with questions.

@punchagan have done this, can you check now ? I range the migrations on my local DB, the match names were generated ;)